src/App.jsx

```diff
@@ -14,70 +17,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedProduct, setSelectedProduct] = useState('Jam');
+
+  const correctTitle = selectedProduct
+    ? `${selectedProduct} is selected`
+    : 'No goods selected';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const ClearButton = () => (
+    <button
+      data-cy="ClearButton"
+      type="button"
+      className="delete ml-3"
+      onClick={() => (setSelectedProduct(''))}
+    />
+  );
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const clearButtonVisibility = selectedProduct
+    ? ClearButton()
+    : '';
```

Comment:
I think readability would be better with component syntax.
```suggestion
  const clearButtonVisibility = selectedProduct
    ? <ClearButton />
    : '';
```